### PR TITLE
Add 爱普云 to showcase

### PR DESCRIPTION
**Pull request by cpunion**

### CODE REVIEW

**Comment by facebook-github-bot-9** [comment]
By analyzing the blame information on this pull request, we identified @brentvatne and @manonthemat to be potential reviewers.

**Comment by facebook-github-bot-9** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -1028,6 +1028,13 @@ var apps = [
     link: 'https://emberall.com/',
     author: 'Kyle Corbitt',
   },
+  {
+    name: '爱普云',
+    icon: 'http://img.applean.cn/icon1024.png',
+    linkAppStore: 'https://itunes.apple.com/cn/app/ai-pu-yun/id1100119649?mt=8',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=cn.applean.apprunner',
+    author: 'Applean.cn'
```
Comment (dsibiski) [inline website/src/react-native/showcase.js]: Please add a trailing comma here.

**Comment by facebook-github-bot-1** [comment]
@cpunion updated the pull request.

**Comment by dsibiski** [comment]
Please sign the CLA and I can merge 👍

**Comment by cpunion** [comment]
Done. Thanks!

**Comment by cpunion** [comment]
Already signed CLA, please update the status.

**Comment by dsibiski** [comment]
@mkonicek Is there a way I can check that it is indeed signed? The @facebook-github-bot doesn't seem to be reporting it right now...

**Comment by cpunion** [comment]
@dsibiski My fault, sign now. (I thought reply is sign)

**Comment by cpunion** [comment]
I have already signed the CLA, please check it. Thanks.

**Comment by lacker** [comment]
Hey, I'm really sorry for the churn, but we are changing up how the showcase works, and just focusing on apps that have some media presence or technical blog posts / other content. So I appreciate the spirit behind this pull request but I think we should close it.

**Pull request closed without merging**